### DISCUSSION TITLE
NFAuthentication.key file not visible in Kodi 20.

### DISCUSSION THREAD

**Discussion by cichytom** [CastagnaIT/plugin.video.netflix#1637] [Q&A] [answered]
Hello, 
In Kodi 20.2, when I go to the directory where the NFAuthentication.key file is located, it is not visible and therefore cannot be selected. This file is definitely visible in the directory I indicate. I checked it using the "File Commander Manager & Vault" application. I have Kodi version 20.2 installed on "Realme TV Stick 4K" with Android TV 11 and on "Google Chromecast 4.0 4K" with Android TV 12 and both devices have the same problem. 
Kodi version: Nexus 20.2
Device 1: Realme Smart TV Stick (Android TV 11)
Device 2: Google Chromecast 4K (Android TV 12)
Add-on v1.22.3

**Comment by CastagnaIT** [comment, marked as answer]
see android permissions: https://kodi.wiki/view/HOW-TO:Install_Kodi_for_Android
